scripts/trustpilot.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs')
+const path = require('path')
+require('dotenv').config();
+
+const fetchTrustpilotData = async () => {
+    const fileDir = path.resolve(__dirname, '../src/data')
+    const fileName = 'trustpilot.json'
+    const filePath = path.join(fileDir, fileName)
+ 
+    try {
+        const appName = 'deriv.com'
+        const apiKey = process.env.TRUSTPILOT_API_KEY
+     
+        if (!appName || !apiKey) {
+            throw new Error('Trustpilot app name or API key is missing')
+        }
+
+        const url = `https://api.trustpilot.com/v1/business-units/find?name=${appName}&apikey=${apiKey}`
+        const response = await fetch(url)
+
+        if (!response.ok) {
+            throw new Error(`Network response was not ok: ${response.statusText}`)
+        }
+
+        const result = await response.json()
+        const tpData = {
+            stars: result.score?.stars || 0,
```

Comment:
Instead of 0, can we have null, in case the api fails we can show a TP image maybe?

Reply:
No @habib-deriv this works as a data. We have a default file the contains static data. If fetching fails that data file won't change and thus will always show trustpilot regardless of the result of this fetch call.